build.cmd

```diff
@@ -107,10 +107,12 @@ if not %CONFIGURATION% == Release goto :eof
 rem Find InnoSetup
 
 if not defined INNOSETUP for %%a in (ISCC.exe) do if not [%%~$PATH:a] == [] set INNOSETUP="%%~$PATH:a"
-if not defined INNOSETUP if exist "%ProgramFiles%\Inno Setup 5\ISCC.exe" set INNOSETUP="%ProgramFiles%\Inno Setup 5\ISCC.exe"
-if not defined INNOSETUP if exist "%ProgramFiles(x86)%\Inno Setup 5\ISCC.exe" set INNOSETUP="%ProgramFiles(x86)%\Inno Setup 5\ISCC.exe"
-if not defined INNOSETUP if exist "%ProgramW6432%\Inno Setup 5\ISCC.exe" set INNOSETUP="%ProgramW6432%\Inno Setup 5\ISCC.exe"
-if not defined INNOSETUP if exist "%SystemDrive%\Inno Setup 5\ISCC.exe" set INNOSETUP="%SystemDrive%\Inno Setup 5\ISCC.exe"
+for %%v in (5 6) do (

```

Comment:
Have you tested Inno Setup 6 and found it to work perfectly (installation, installation of 32-bit on 64-bit, uninstallation, signing)?

Reply:
I've used both install/uninstall on 64-bit on 64-bit.
I can test 32-on-64.
IDK about signing.

Reply:
Great, thank you.